src/MacVim/MMWindowController.m

```diff
@@ -1329,6 +1333,26 @@ - (void)windowDidFailToExitFullScreen:(NSWindow *)window
 
 #endif // (MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_7)
 
+- (void)runAfterWindowPresentedUsingBlock:(void (^)(void))block
+{
+    if (windowPresented) { // no need to defer block, just run it now
+        block();
+        return;
+    }
+
+    // run block later
+    if (afterWindowPresentedQueue == nil) {
+        afterWindowPresentedQueue = [[NSMutableArray alloc] init];
+    }
+    [afterWindowPresentedQueue addObject:[NSBlockOperation blockOperationWithBlock:block]];
+}
+
+- (void)processAfterWindowPresentedQueue
```

Comment:
Could you move `processAfterWindowPresentedQueue` to Private category and add declaration to align with other private methods?